src/OpenLoco/Audio/Audio.h

```diff
@@ -61,10 +61,9 @@ namespace OpenLoco::Audio
 
     enum class ChannelId
     {
-        bgm,
+        music,
         unk_1,
         ambient,
-        title,
```

Comment:
this might cause issues with the hooks. Might be an idea to leave an unused entry.